.pep8

```diff
@@ -0,0 +1,4 @@
+[pep8]
```

Comment:
Reviewers: I borrowed this from the ecommerce .pep8 file:

https://github.com/edx/ecommerce/blob/master/.pep8